src/Aurora.Scrapers/Extensions/StringExtensions.cs

```diff
@@ -1,6 +1,23 @@
-﻿namespace Aurora.Scrapers.Extensions;
+﻿using System.Text.RegularExpressions;
+
+namespace Aurora.Scrapers.Extensions;
 
 internal static class StringExtensions
 {
     public static string FormatTermToUrl(this string term) => term.Replace(" ", "+");
+
+    private static Regex _numbersRegex = new Regex("\\d+(\\.\\d+)?", RegexOptions.Multiline | RegexOptions.Compiled);
```

Comment:
did u say regex bad?

Reply:
If we ever are interested in improving performance, then we might look into optimizing that, but it is good enough with this regex